pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -321,14 +328,27 @@ export class ConfigurationCard extends React.Component {
     }
 
     render() {
+        // We use a Privileged component for its ability to
+        // conditionally show a tooltip, even when the button is not
+        // actually disabled, so the "allowed" property really means
+        // "does not have tooltip" here.
+
+        const hostname_tooltip = (this.permission.allowed == false
+            ? cockpit.format(_("The user $0 is not permitted to modify hostnames"),
+                             this.permission.user ? this.permission.user.name : '')
+            : this.realmd.hostname_button_tooltip);
+        const hostname_disabled = this.permission.allowed == false || this.realmd.hostname_button_disabled;
+
         const hostname_button = (
-            <PrivilegedButton variant="link" buttonId="system_information_hostname_button"
```

Comment:
Perhaps we should teach PrivilegedButton to have an additional disabled reason -- I think on the Storage page you also have that case, right? I'm happy to help out with this, given that I wrote this component.

Reply:
We also need the "tooltip but enabled" case here, so using long-hand with both Privileged and Button is probably best.